modules/Movecraft/src/main/java/net/countercraft/movecraft/async/translation/TranslationTask.java

```diff
@@ -393,6 +394,12 @@ else if (world.equals(craft.getWorld())
 
         updates.add(new CraftTranslateCommand(craft, new MovecraftLocation(dx, dy, dz), world));
 
+        //translates the craft's tracked locations.
+        Iterator<TrackedLocation> trackedLocations = craft.getTrackedLocations().values().iterator();
+        while(trackedLocations.hasNext()) {
+            trackedLocations.next().translate(dx, dy, dz);
+        }
+
```

Comment:
Why don't they need translation? Aren't all the MovecraftLocations absolute locations in the world? Or is that just not true?

Reply:
`MovecraftLocation`s are just a compact representation of an x,y,z vector.  Most often they are used to represent an absolute world position (in hitboxes) but nothing prevents us from using them as an offset.  In actuality, it would make the most sense to use the `offset`'s hash as the hash of the `TrackedLocation` to keep the `HashSet` performant.